lib/APINotes/APINotesWriter.cpp

```diff
@@ -475,10 +475,16 @@ namespace {
   template<typename T>
   void emitVersionedInfo(
          raw_ostream &out,
-         const SmallVectorImpl<std::pair<VersionTuple, T>> &infoArray,
+         SmallVectorImpl<std::pair<VersionTuple, T>> &infoArray,
          llvm::function_ref<void(raw_ostream &out,
                                  const typename MakeDependent<T>::Type& info)>
            emitInfo) {
+    std::sort(infoArray.begin(), infoArray.end(),
```

Comment:
No `array_pod_sort`, eh?

Reply:
It's not necessarily POD, is it?

Reply:
Well. Maybe not by the language definition., but it'd work. `std::sort()` is not exactly a small thing to instantiate.